src/app/organization/password-preferences/password-preferences.component.ts

```diff
@@ -0,0 +1,79 @@
+/** Angular Imports */
+import { Component, OnInit } from '@angular/core';
+import { FormGroup, FormBuilder } from '@angular/forms';
+import { Router, ActivatedRoute } from '@angular/router';
+
+/** Custom Services */
+import { OrganizationService } from '../organization.service';
+
+/**
+ * Password preferences component.
+ */
+@Component({
+  selector: 'mifosx-password-preferences',
+  templateUrl: './password-preferences.component.html',
+  styleUrls: ['./password-preferences.component.scss']
+})
+export class PasswordPreferencesComponent implements OnInit {
+
+  /** Password preferences form. */
+  passwordPreferencesForm: FormGroup;
+  /** Password preferences data. */
+  passwordPreferencesData: any;
+
+  /**
+   * Retrieves the password preferences data from `resolve`.
+   * @param {FormBuilder} formBuilder Form Builder.
+   * @param {OrganizationService} organizationService Organization Service.
+   * @param {ActivatedRoute} route Activated Route.
+   * @param {Router} router Router for navigation.
+   */
+  constructor(private formBuilder: FormBuilder,
+              private organizationService: OrganizationService,
+              private route: ActivatedRoute,
+              private router: Router) {
+    this.route.data.subscribe((data: { passwordPreferencesTemplate: any}) => {
+      this.passwordPreferencesData = data.passwordPreferencesTemplate;
```

Comment:
I have passed through the 'template' of the password preferences but really this is the actual data, so I named it as passwordPreferencesData here. However, I have named it as 'template' in the resolver, service and routing to ensure that the non-template API endpoint can also be implemented and to follow the convention for consistency.